Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
ignores meaning returns error or keeps everything as is? I'm trying to avoid the confusion when ARM shows the argument set that doesn't match what actually was applied. 

As well as a general confusion of applying ARM update without visible effect.